editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -98,19 +101,26 @@ const FeedbackComponent = (
             />
           </Grid>
         </StyledToggleButtonGroup>
-        {/* </InputLabel> */}
-        <InputLabel label="Please tell us more about your experience">
-          <RichTextInput
-            name="feedback"
-            value={formik.values.feedback}
-            onChange={formik.handleChange}
+        {props.freeformQuestion && (
+          <InputLabel
+            label={
+              <ReactMarkdownOrHtml
+                source={props.freeformQuestion}
+                id={"RATING_QUESTION"}
+                openLinksOnNewTab
+              />
+            }
           />
-        </InputLabel>
+        )}
+        <RichTextInput
+          name="feedback"
+          value={formik.values.feedback}
+          placeholder="What did you think?"
+          onChange={formik.handleChange}
+        />
```

Comment:
The `RichTextInput` is reserved for editor modals, I'd take a look at the `@planx/components/TextInput` Public component files for examples of what we expect for the public display here ! 

(Feel like this points to bigger organisation thing around it being easy to confuse which components are for what currently - let's keep this in mind as we continue to think about how to better organise files!)